### PR TITLE
Properly enforced LF in DMAPI Updater

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This PR makes LF properly enforced in the DMAPI updater action

## Why It's Good For The Repository
Auto updates shouldnt be failing CI instantly

## Changelog
N/A

### CODE REVIEW

**Comment by AffectedArc07** [comment]
To aid reviewing:
image
Disable whitespace changes and it makes the other files minimise since they had no actual diff

File: .github/workflows/update_tgs_dmapi.yml [hunk 1]
```diff
@@ -19,12 +19,20 @@ jobs:
         git checkout tgs-dmapi-update
         git reset --hard master
 
+    - name: Install Deps
+      run: sudo apt install dos2unix
+
```
Comment (dearmochi) [inline .github/workflows/update_tgs_dmapi.yml:24]: Undocumented change?

Comment (AffectedArc07) [reply .github/workflows/update_tgs_dmapi.yml:24]: This is what enforces the LF since it force converts it all

Comment (dearmochi) [reply .github/workflows/update_tgs_dmapi.yml:24]: Fair enough

**Review by dearmochi — APPROVED** [review]
LGTM, even if I'm not familiar with the infrastructure

**Review by farie82 — APPROVED** [review]
Should be fine if it's tested and confirmed to work

**Pull request merged**